scipy/optimize/_qap.py

```diff
@@ -0,0 +1,413 @@
+import numpy as np
+import math
+from . import linear_sum_assignment, minimize_scalar, OptimizeResult
+
+
+def quadratic_assignment(
+    cost_matrix,
+    dist_matrix,
+    seed=np.array([[], []]).T,
```

Comment:
Change the default to `None` here and in the docstring.
In the code, you can `if seed is None: seed =` whatever, but it is unusual to see a specific empty array like this as a default.